src/CMakeLists.txt

```diff
@@ -1,8 +1,26 @@
 set(TARGET_EXE aktualizr-lite)
 set(TARGET ${MAIN_TARGET_LIB})
 
-set(SRC helpers.cc liteclient.cc composeappmanager.cc docker.cc composeapp.cc composeapptree.cc ostree/sysroot.cc ostree/repo.cc)
-set(HEADERS helpers.h liteclient.h composeappmanager.h docker.h composeapp.h composeapptree.h ostree/sysroot.h ostree/repo.h)
+set(SRC helpers.cc
+        composeappengine.cc
+        appengine.cc
```

Comment:
this is an empty file. is a place holder or something ?

Reply:
yes, I can remove it.